vendor/k8s.io/utils/trace/trace.go

```diff
@@ -83,37 +102,24 @@ func (t *Trace) Log() {
 }
 
 func (t *Trace) logWithStepThreshold(stepThreshold time.Duration) {
-	var buffer bytes.Buffer
 	tracenum := rand.Int31()
 	endTime := time.Now()
 
 	totalTime := endTime.Sub(t.startTime)
-	buffer.WriteString(fmt.Sprintf("Trace[%d]: %q ", tracenum, t.name))
-	if len(t.fields) > 0 {
-		writeFields(&buffer, t.fields)
-		buffer.WriteString(" ")
-	}
-	buffer.WriteString(fmt.Sprintf("(started: %v) (total time: %v):\n", t.startTime, totalTime))
+	klog.InfoS("Trace start", logTraceArgs(tracenum, t, totalTime)...)
```

Comment:
Think to consider: depending on klog verbosity we will get two outputs

if `klog.V(4).Enabled()`:
```
Trace start ...
Trace step ...
Trace stop ...
```
else
```
Trace start
```

It could be confusing for users that there is a `Trace start` message without `Trace end`.
I see two options
* Change `Trace start` to `Trace`
* depending on `klog.V(4).Enabled()` change message

Reply:
I think option 1 would make more sense.

Reply:
I think option1 is good. I modify this.